packs/data/pfs-season-4-bestiary.db/depleted-roiling-incant.json

```diff
@@ -0,0 +1,905 @@
+                },
+                "trigger": {
+                    "value": ""
+                },
+                "weapon": {
+                    "value": ""
+                }
+            },
+            "type": "action"
+        },
+        {
+            "_id": "hfbHiFnMOX2vWG3b",
+            "flags": {
+                "core": {
+                    "sourceId": "Compendium.pf2e.bestiary-ability-glossary-srd.zU3Ovaet4xQ5Gmvy"
+                }
+            },
+            "img": "systems/pf2e/icons/actions/TwoActions.webp",
+            "name": "Engulf",
+            "sort": 900000,
+            "system": {
+                "actionCategory": {
+                    "value": "offensive"
+                },
+                "actionType": {
+                    "value": "action"
+                },
+                "actions": {
+                    "value": 2
+                },
+                "description": {
+                    "value": "<p>@Check[type:reflex|dc:25], [[/r {2d6}[force]+{3d4}[fire]]]{2d6 force damage plus 3d4 fire damage}, @UUID[Compendium.pf2e.actionspf2e.Escape]{Escape} DC 25, Rupture 15</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.Engulf]</p>"
+                },
+                "requirements": {
+                    "value": ""
+                },
+                "rules": [],
+                "slug": "engulf",
+                "source": {
+                    "value": "Pathfinder Bestiary"
+                },
+                "traits": {
+                    "custom": "",
+                    "rarity": "common",
+                    "value": []
+                },
+                "trigger": {
+                    "value": ""
+                },
+                "weapon": {
+                    "value": ""
+                }
+            },
+            "type": "action"
+        },
+        {
+            "_id": "M6HiBc42J8BI5rZo",
+            "img": "systems/pf2e/icons/actions/Passive.webp",
+            "name": "Unstable Magic",
+            "sort": 1000000,
+            "system": {
+                "actionCategory": {
+                    "value": "offensive"
+                },
+                "actionType": {
+                    "value": "passive"
+                },
+                "actions": {
+                    "value": null
+                },
+                "description": {
+                    "value": "<p>A roiling incant is as much a mass of unstable magic as it is a creature. Is isn't living or undead, nor is it even a construct. It can't be healed or @UUID[Compendium.pf2e.actionspf2e.Repair]{Repaired} and is destroyed at 0 Hit Points, though it naturally recovers a number of Hit Points equal to its level × its Constitution modifier ([[/r {42}[healing]]]{42} for most depleted roiling incants) each day.</p>\n<p>Each time a roiling incant casts one of its non-cantrip spells, it drains its own magic to do so, taking [[/r {5}[force]]]{5 force damage}.</p>"
```

Comment:
Huh, we may need to go through and fix the mode of living for these things and any other weird one offs